### DISCUSSION TITLE
Poetry looking for `traitlets 5.2.2.post0`

### DISCUSSION THREAD

**Discussion by adam-grant-hendry** [python-poetry/poetry#6825] [Q&A]
[`commitizen`](https://commitizen-tools.github.io/commitizen/) is encountering [a strange issue in its CI workflows](https://github.com/commitizen-tools/commitizen/actions/runs/3266510247/jobs/5370341870) that `poetry` cannot find a nonexistent package:

```
Package traitlets (5.2.2.post0) not found
```

`traitlets` went from `5.2.2` -> `5.2.2.post1`, but never released a `post0` (nor was a `post0` ever yanked). See the [PyPI history](https://pypi.org/project/traitlets/#history).

The workflow doesn't cache virtual environments, so there is no cache to manually clear on GitHub.

Has anyone experienced a similar issue? Is the dependency resolver expriencing an issue here because `traitlets` uses an extra `post` identifier in its versions and `poetry` is assuming there should be a `post0`...?

Any help is welcome and would be appreciated! Thank you!

**Comment by neersighted** [comment]
This is odd -- it sounds like it might be a Poetry bug, so I've created an issue from your discussion. Please post your pyproject.toml and versions (OS, Python, Poetry) as you would following the issue template when you have a minute.
